### ISSUE TITLE
Not compatible with latest version

### ISSUE THREAD

**Issue by SparklingSoftware** [tweichart/JSON-API-for-BuddyPress#1]
Hi,

First of all: Awesome plugin!! This is exactly what I need!

However, it doesn't seem to work with the current version. It's saying "BuddyPress" needs to be installed. It looks like it's trying to find a particular version of buddypress perhaps?

Happy to fork and have a look, but it will be a few weeks before I really get around to needing/using this...

Cheers,
Stephan

**Comment by tweichart** [comment]
Thanks for your comment ;-)
I'll try to look in the problem asap, I'll keep you updated.

**Comment by mastef** [comment]
What about 1.7 support? Is it compatible?

**Comment by tweichart** [comment]
it's in my shedule for the next weekend to check and fix it if needed ;-)

**Comment by tweichart** [comment]
so I've checked it and it works for BuddyPress 1.7, although I had to deactivate the json api and my plugin to get the update running and reactivate it afterwards. But despite that it works ;-)

Don't know where's the problem in the first post, it works with BuddyPress 1.7 and Wordpress 3.5.1. Maybe there are more infos?

**Comment by SparklingSoftware** [comment]
Hmm.. Let me go back and retry, perhaps its a human error?? :-)
If not, I'll try to give you as much info as I can and start debugging myself.

Stephan

**Comment by SparklingSoftware** [comment]
Sorry, still the same issue.

In the "Admin" -> "Plugins" area, after I activated your "JSON API for Buddypress" plugin (version 1.0) I get the following error message (Red box at the top of the page):

JSON API for Buddypress requires the BuddyPress plugin to be activated. Please install / activate BuddyPress first, or deactivate JSON API for Buddypress.

And more annoyingly, the plugin doesn't seem to work :-(
As I said, I'll also try to determine what the cause is.

Stephan

**Comment by tweichart** [comment]
I've merged your changes. Could you please tell me what exactly doesn't work? Does the whole plugin fail or is is it just some functions? Can you access the json-api methods?

**Comment by SparklingSoftware** [comment]
Hi Tobias,

It works now,

I didn't realize that: When you spoke about "JSON-API" you meant a per-requisite plugin. So it turned out to be a RTFM issue. In my defense though, the error message send me off on a ghost chase a bit. I did get very confused that there wasn't any trace of the admin config anywhere to be found though.

Anyway, I've reverted back half of the change I've send you, leaving the fix for the alert text in place so others (who also don't read the manual) won't have the same issue...

Awesome plugin,
Stephan
